x/feeburner/types/params.go

```diff
@@ -0,0 +1,149 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"gopkg.in/yaml.v2"
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
+
+var (
+	KeyNeutronDenom     = []byte("NeutronDenom")
+	DefaultNeutronDenom = "stake"
+	KeyGaiaDenom        = []byte("GaiaDenom")
+	DefaultGaiaDenom    = "uatom"
+	KeyTreasuryAddress  = []byte("TreasuryAddress")
+	// XXX: that's just a random address, I think there should be something better for a default
+	DefaultTreasuryAddress = "neutron12m0f7r5wzuskcuzrreppjgsu8fvzkqced7uevn"
```

Comment:
Treasury should be added in the genesis, so we need to put here genesis address of the treasury contract. We need to have instructions for genesis generation, or update `init.sh` to get contract address from there if it is possible.